src/postgres_mcp/dta/safe_sql.py

```diff
@@ -551,11 +713,154 @@ class SafeSqlDriver(SqlDriver):
         NamedArgExpr,
         RangeFunction,
         A_ArrayExpr,
+        # Window functions
+        WindowFunc,
+        WindowDef,
+        WindowClause,
+        # Table functions
+        TableFunc,
+        RangeTableFunc,
+        RangeTableFuncCol,
+        # Array access
+        A_Indirection,
+        A_Indices,
+        # GROUPING SETS, CUBE, ROLLUP
+        GroupingSet,
+        GroupingFunc,
+        # Table sampling
+        RangeTableSample,
+        TableSampleClause,
+        # Row comparison
+        RowCompareExpr,
+        # Collation
+        CollateClause,
+        # Others
+        TargetEntry,
+        # Additional expression types
+        ScalarArrayOpExpr,  # ARRAY[...] @> or <@ operators
+        NotifyStmt,  # NOTIFY command
     }
 
     ALLOWED_EXTENSIONS: ClassVar[set[str]] = {
+        # Core PostgreSQL extensions
         "hypopg",
         "pg_stat_statements",
+        "pg_trgm",
```

Comment:
Question: does it make sense to support creating an extension when we have not reviewed the functions that it provides?